src/applications/vaos/referral-appointments/utils/referrals.js

```diff
@@ -10,82 +9,68 @@ const expiredUUIDBase = '445e2d1b-7150-4631-97f2-f6f473bdef';
  */
 
 const createReferralListItem = (
-  startDate,
-  uuid,
-  providerId = '111',
   expirationDate,
+  uuid,
   categoryOfCare = 'Physical Therapy',
 ) => {
-  const [year, month, day] = startDate.split('-');
+  const [year, month, day] = expirationDate.split('-');
   const relativeDate = new Date(year, month - 1, day);
   const mydFormat = 'yyyy-MM-dd';
   return {
-    uuid,
-    referralDate: startDate,
-    categoryOfCare,
-    expirationDate:
-      expirationDate || format(addMonths(relativeDate, 6), mydFormat),
-    providerId,
+    attributes: {
+      expirationDate:
+        expirationDate || format(addMonths(relativeDate, 6), mydFormat),
+      uuid,
+      categoryOfCare,
+    },
   };
 };
 
 /**
- * Creates a referral object relative to a start date.
+ * Creates a referral object with specified uuid and expiration date.
  *
- * @param {String} startDate The date in 'yyyy-MM-dd' format to base the referrals around
  * @param {String} uuid The UUID for the referral
- * @param {String} providerId The ID for the provider
  * @param {String} expirationDate The date in 'yyyy-MM-dd' format to expire the referral
  * @returns {Object} Referral object
  */
 const createReferralById = (
-  startDate,
   uuid,
-  providerId = '111',
   expirationDate,
   categoryOfCare = 'Physical Therapy',
 ) => {
-  const [year, month, day] = startDate.split('-');
-  const relativeDate = new Date(year, month - 1, day);
-
+  // if no expiration date is provided, set it to 6 months from today
+  const today = new Date();
   const mydFormat = 'yyyy-MM-dd';
-
   return {
-    uuid,
-    referralDate: format(relativeDate, mydFormat),
-    expirationDate:
-      expirationDate || format(addMonths(relativeDate, 6), mydFormat),
-    referralNumber: 'VA0000009880',
-    status: 'Approved',
-    categoryOfCare,
-    stationId: '528A4',
-    sta6: '534',
-    referringFacility: 'Batavia VA Medical Center',
-    referringFacilityInfo: {
-      facilityName: 'Batavia VA Medical Center',
-      facilityCode: '528A4',
-      description: 'Batavia VA Medical Center',
-      address: {
-        address1: '222 Richmond Avenue',
-        city: 'BATAVIA',
-        state: 'NY',
-        zipCode: '14020',
+    id: uuid,
+    type: 'referral',
+    attributes: {
+      uuid,
+      expirationDate: expirationDate || format(addMonths(today, 6), mydFormat),
+      referralNumber: 'VA0000009880',
+      categoryOfCare,
+      referringFacilityInfo: {
+        facilityName: 'Batavia VA Medical Center',
+        facilityCode: '528A4',
+        address: {
+          address1: '222 Richmond Avenue',
+          city: 'BATAVIA',
+          state: 'NY',
+          zipCode: '14020',
+        },
+        phone: '(585) 297-1000',
       },
-      phone: '(585) 297-1000',
-    },
-    referralStatus: 'open',
-    provider: {
-      id: providerId,
-      name: 'Dr. Moreen S. Rafa',
-      location: 'FHA South Melbourne Medical Complex',
+      provider: {
```

Comment:
I don't think you care here, but I have more info for the provider if you need it
```json
"provider": {
                "name": "A & D HEALTH CARE PROFS",
                "npi": "1346206547",
                "telephone": "(937) 236-6750",
                "location": "A & D HEALTH CARE PROFS"
            },
```